setup.py

```diff
@@ -242,6 +243,9 @@ def _run_domain_js(self, domain):
         'distutils.commands': [
             'build_sphinx = sphinx.setup_command:BuildDoc',
         ],
+        'pytest11': [
```

Comment:
Note for me: this entrypoint is described in pytest docs.
https://docs.pytest.org/en/6.2.x/writing_plugins.html#making-your-plugin-installable-by-others